src/storage/patch/PatchHandler.ts

```diff
@@ -1,6 +1,15 @@
 import type { Patch } from '../../ldp/http/Patch';
 import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
 import { AsyncHandler } from '../../util/handlers/AsyncHandler';
+import type { ResourceStore } from '../ResourceStore';
 
-export abstract class PatchHandler
-  extends AsyncHandler<{identifier: ResourceIdentifier; patch: Patch}, ResourceIdentifier[]> {}
+export type PatchHandlerArgs = {
```

Comment:
Could you make them generic over `T implements ResourceStore = ResourceStore`?

That way, we can write patchers for specific kind of stores.
Which was one of the reasons to use the store as a constructor argument.